ArduCopter/AP_Arming.cpp

```diff
@@ -596,6 +596,21 @@ bool AP_Arming_Copter::arm_checks(AP_Arming::Method method)
         return false;
     }
 
+    // Do not arm if EKF lane 1 is not used as primary during arming, when using EK2.
```

Comment:
Should the last word in the comment be `EKF2`?

Reply:
That was deliberate as I kept it consistent with how we name the parameters corresponding to each EKF (for example EK2_ENABLE), but yea I will change it to EKFx, rather than EKx to make it more readable.